lib/bgsite_official/home/websites.ex

```diff
@@ -30,7 +30,9 @@ defmodule BgsiteOfficial.Home.Websites do
     wildcard_search = "%#{search_term}"
```

Comment:
Let's check if there is a `search_term` at all, and in case of empty `search_term` to fallback to `all websites` on this action. Now, this function searches for titles and descriptions similar to empty criteria.